src/js/handleGame.js

```diff
@@ -0,0 +1,164 @@
+import createGameBoards from "./boardView.js";
+import {UltimateBoard} from "./Boards/UltimateBoard.js";
+import {HumenPlayer} from "./Players/HumenPlayer.js";
+import {ComputerPlayer} from "./Players/ComputerPlayer.js";
+import { Board } from "./Boards/Board.js";
+
+const   boardSize = 3,
+        elementsID = {gameStatus: 'gameStatus',
+                    humPlayersign: 'humPlayerSign',
+                    compPlayersign: 'compPlayerSign'};
+let humPlayer,
+    compPlayer,
+    currPlayer,
+    bigBoard,
+    scoreBrd;
+
+
+export function startGame(mainBoardElem, scoreElem, gameInfoElem){
+    initGameBoards(mainBoardElem, scoreElem);
+    initPlayers();
+    initGameInfo(gameInfoElem);
+    updateCurrPlayer();
+}
+
+function initGameBoards(mainBoardElem, scoreElem){
+    // Creating all game boards
+    const brds = createGameBoards(boardSize);
+
+    mainBoardElem.appendChild(brds.mainBrd.tblElem);
+    scoreElem.appendChild(brds.scoreBrd.tblElem);
+
+    bigBoard = new UltimateBoard("brd-0", boardSize, brds.mainBrd.tblElem, brds.mainBrd.cellsArr);
+    scoreBrd = new Board("scrBrd", boardSize, brds.scoreBrd.tblElem, brds.scoreBrd.cellsArr);
+
+    addClickEventCells();
+}
+
+function initPlayers(plyr1, plyr2){
+    humPlayer = new HumenPlayer("You", "X");
+    compPlayer = new ComputerPlayer("Comp", "O");
+}
+
+function initGameInfo(gameInfoElem){
+    // const elem =  document.createElement("p");
+    //.setAttribute();
+
+    $(`#${elementsID.humPlayersign}`).html(`<p>Player 1: ${humPlayer.sign}</p>`);
+    $(`#${elementsID.compPlayersign}`).html(`<p>Player 2: ${compPlayer.sign}</p>`);
+}
+
+function updateCurrPlayer(){
+    if (currPlayer === humPlayer){
+        currPlayer = compPlayer;
+    } else {
+        currPlayer = humPlayer;
+    }
+    $(`#${elementsID.gameStatus}`).text(`Turn of: ${currPlayer.name}`);
+
+    if (currPlayer instanceof ComputerPlayer){
+        compMove();
+    }
+}
+
+function addClickEventCells(){
+    bigBoard.boardElem.addEventListener("click", cellClick);
+}
+
+function removeEvents(){
+    bigBoard.boardElem.removeEventListener("click", cellClick);
+}
+
+// Handlinng cell click event
+function cellClick(event){
+    if (currPlayer instanceof ComputerPlayer){
+        alert("Not your turn!");
+        return;
+    }
+
+    const clsSt = event.target.closest('.innerCell');
+
+    // If its not an inner cell that clicked
+    if (clsSt === null){
+        return;
+    }
+
+    playMove(clsSt.id);
+}
+
+// Extracting from cellId its positions in tables
```

Comment:
I suggested to introduce a class for cell id, now I'll suggest to give it methods "toString" if you need a string representation really, and static "parse" method to parse a string into cell id

Reply:
If not clear, the string representation for cell id should be used little as possible throughout the code